medusa/tv/episode.py

```diff
@@ -974,6 +975,15 @@ def delete_episode(self):
 
         raise EpisodeDeletedException()
 
+    def get_manual_search_results(self, manual_search_type='episode', show_all_results=0, perform_search=0):
```

Comment:
This should be returning all releases for an episode or season. This should have no coupling to manual search as it should also return cached results from other types of searches like daily, etc.